opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -44,6 +44,35 @@ message InstrumentationLibraryMetrics {
   repeated Metric metrics = 2;
 }
 
+// A representation of raw measurements, which can have statistical meaning based
+// on how the measurement was sampled, as well as a span ID and trace ID of the
+// active span when the measurement was recorded
+message RawValue {
```

Comment:
Would be good to be consistent for the ordering with data points, so maybe labels first, etc.

Reply:
fixed